connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnectorTest.java

```diff
@@ -59,12 +59,13 @@ public void testMirrorCheckpointConnectorDisabled() {
 
         Set<String> knownConsumerGroups = new HashSet<>();
         knownConsumerGroups.add(CONSUMER_GROUP);
+        assertMirrorCheckpointConnectorDisabled(new MirrorCheckpointConnector(knownConsumerGroups, config));
+    }
+
+    private void assertMirrorCheckpointConnectorDisabled(MirrorCheckpointConnector connector) {
```

Comment:
Is this change still necessary? Looks like it might have been left over from a previous draft?

Reply:
removed